nautobot/extras/tables.py

```diff
@@ -267,6 +268,34 @@ class Meta(BaseTable.Meta):
         )
 
 
+class DynamicGroupTable(BaseTable):
+
+    pk = ToggleColumn()
+    name = tables.LinkColumn(args=[Accessor("slug")])
```

Comment:
`LinkColumn` is deprecated - this *should* be able to just be:
```suggestion
    name = tables.Column(linkify=True)
```